src/services/firebase.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import firebase from "firebase"
+import StyleFirebaseAuth from "react-firebaseui/StyledFirebaseAuth"
+
+firebase.initializeApp({
+   apiKey: "AIzaSyAXDqsWK4quNVaf9-YV2e28NsxkfA9rzJA",
+   authDomain: "dreamerscholars.firebaseapp.com"
+})
+
+class FirebaseAuth extends Component {
+   state = { isSignedIn: false }
+   uiConfig = {
+      signInFlow: "popup",
+      signInOptions: [
+         firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+         firebase.auth.EmailAuthProvider.PROVIDER_ID,
+         firebase.auth.FacebookAuthProvider.PROVIDER_ID
+      ],
+      callbacks: {
+         signInSuccess: () => false
+      }
+   }
```

Comment:
If this UI config is constant, then I suggest extracting it as a constant in this file (maybe `FIREBASE_UI_CONFIG`) and exporting it so other files can access it:
```js
export const FIREBASE_UI_CONFIG;
```

Reply:
Yeah that needs to occur because other files need to access it to know if the user is logged in or not. How can I export a const function? Ill read about it in the mean time see if I can figure a way to do this